x-pack/plugins/infra/server/lib/infra_ml/queries/common.ts

```diff
@@ -44,6 +44,7 @@ export const createTimeRangeFilters = (startTime: number, endTime: number) => [
       timestamp: {
         gte: startTime,
         lte: endTime,
+        format: 'epoch_millis',
```

Comment:
This is called when showing the anomalies of of a node in the Metrics waffle chart